templateBuilder.go

```diff
@@ -105,6 +105,11 @@ func getTemplates(params Params) []string {
 	if (params.Kind == "deployment" || params.Kind == "statefulset") && (params.Visibility == "private" || params.Visibility == "iap" || params.Visibility == "public-whitelist") {
 		templatesToMerge = append(templatesToMerge, "ingress.yaml")
 	}
+
+	if params.Kind == "deployment" && params.Visibility == "apigee" {
+		templatesToMerge = append(templatesToMerge, "ingress-apigee.yaml")
```

Comment:
@JorritSalverda Do you suggest to include ingress-office and ingress-internal as well to be able to test internally?

Reply:
I think that adding `ingress-office` will lead to dns conflicts where multiple ingresses try to set the same dns record. The internal ingress can be added at the same time. If applications want to prevent traffic from any other source they'll have to block it themselves. With IAP for example the application itself still has to check that a certain iap header is sent with any incoming request and is valid.